ctypesgen/ctypedescs.py

```diff
@@ -276,12 +276,8 @@ def __init__(self, restype, parameters, variadic, attrib=dict()):
             self.restype = CtypesPointer(CtypesSpecial("c_ubyte"), ())
             self.errcheck = CtypesPointerCast(CtypesSpecial("c_void_p"))
 
-        # Return "String" instead of "POINTER(c_char)"
-        if self.restype.py_string() == "POINTER(c_char)":
-            if "const" in self.restype.qualifiers:
-                self.restype = CtypesSpecial("c_char_p")
-            else:
-                self.restype = CtypesSpecial("String")
+        if (self.restype.py_string() == "POINTER(c_char)"):
+            self.restype = CtypesSpecial("c_char_p")
```

Comment:
Hmm, according to the [ctypes docs](https://docs.python.org/3.7/library/ctypes.html#ctypes.c_char_p), it's not generally possible to substitute `POINTER(c_char)` with `c_char_p` - only for null-terminated strings, so I guess this clause will have to be removed?

Reply:
Done.